### PR TITLE
Run SwiftLint auto correct command

### PR DESCRIPTION
**Pull request by kishorepran**
run the swiftlint auto correct command
mostly works on removing extra space and formats code in some cases it has also removed extra parenthesis and brackets this is forts step to start improving code quality

### CODE REVIEW

**Review by Suraj1999 — APPROVED** [review]

File: Services/Sources/Services/Camera/CameraService.swift
Comment (kishorepran) [inline Services/Sources/Services/Camera/CameraService.swift:1]: will cover this in next pr

File: Services/Sources/Services/Location/LocationService.swift
Comment (kishorepran) [inline Services/Sources/Services/Location/LocationService.swift:1]: Will cover this in next pr

File: animeal/src/Flows/Profile/Model/Items/ProfileModelAction.swift
Comment (kishorepran) [inline animeal/src/Flows/Profile/Model/Items/ProfileModelAction.swift:1]: Will cover in next pr

File: animeal/src/Flows/Profile/ProfileViewModel.swift
Comment (kishorepran) [inline animeal/src/Flows/Profile/ProfileViewModel.swift:1]: Look like false alarm will cover in next pr

**Review by shankywave1 — APPROVED** [review]

**Pull request merged**